src/components/search/curated/next/index.tsx

```diff
@@ -1,18 +1,72 @@
 import {CardResource} from 'types'
 import {VerticalResourceCard} from 'components/card/verticle-resource-card'
+import groq from 'groq'
+import {VerticalResourceCollectionCard} from 'components/card/vertical-resource-collection-card'
+
+const SearchNext = ({topic}: any) => {
+  console.log('test: ', topic)
+  const location = 'next landing'
+
+  const introCollection = get(topic, 'introCourses')
+  const featuredTalksArticles = get(topic, 'featuredTalksArticles')
+  const advancedConcepts = get(topic, 'advancedConcepts')
+  const portfolioProjects = get(topic, 'portfolioProjects')
 
-const SearchNext = () => {
   return (
-    <SearchCuratedEssential
-      topic={{
-        label: 'Next.js',
-        name: 'next',
-        description: `Next.js gives you the best developer experience with all the features you need for production: hybrid static & server rendering, TypeScript support, smart bundling, route pre-fetching, and more. No config needed.`,
-      }}
-      CTAComponent={EcommerceCTA}
-    />
+    <div>
+      <SearchCuratedEssential
+        topic={{
+          label: 'Next.js',
+          name: 'next',
+          description: `Next.js gives you the best developer experience with all the features you need for production: hybrid static & server rendering, TypeScript support, smart bundling, route pre-fetching, and more. No config needed.`,
+        }}
+        CTAComponent={EcommerceCTA}
+      />
+      <div className="md:col-span-4 col-span-12 flex flex-col">
+        <ThreeLevels
+          beginner={introCollection}
+          intermediate={featuredTalksArticles}
+          advanced={advancedConcepts}
+          location={location}
+        />
+      </div>
+      <section className="md:mt-5 mt-5 grid gap-5 md:bg-gray-100 md:dark:bg-gray-700 rounded-lg md:p-5 relative">
+        <div className="lg:col-span-8 col-span-12 space-y-5 flex flex-col">
+          <header className="py-5 md:px-8 px-5 rounded-md flex md:flex-row flex-col md:text-left text-center md:space-y-0 space-y-3 md:items-start items-center justify-center md:space-x-5 space-x-0">
+            <div className="flex-shrink-0"></div>
+            <div className="max-w-screen-sm space-y-3">
+              <h1 className="md:text-3xl text-2xl dark:text-gray-200 font-bold leading-tight">
+                Building your portfolio!
+              </h1>
+              <div className="leading-relaxed text-gray-700 dark:text-gray-50 space-y-3">
+                <p>
+                  Learn to create your own Next.js project from start to finish!
+                  These courses will take you from initializing your app to
+                  creating your own store with an amazing user interface!
+                </p>
+              </div>
+            </div>
+          </header>
+          <div className="flex flex-col flex-grow">
+            <div className="grid lg:grid-cols-12 grid-cols-1 gap-5 mt-5 flex-grow">
+              {portfolioProjects.resources.map((resource: any) => {
```

Comment:
This is where the issue is Zac. `portfolioProjects` on first load is returning `undefined`. After refreshing, it returns my array. 

I snagged this bit of code from the React landing page and it works perfectly fine over there. The only difference is the react landing page isn't loading this data from sanity.